src/room_util.c

```diff
@@ -532,4 +532,36 @@ void query_room(struct Room *room)
     create_message_box((const char*)&title, name, (const char*)&owner, (const char*)&health, (const char*)&capacity, (const char*)&efficiency);    
 }
 
+void check_for_and_update_surrounding_rooms(MapSlabCoord slb_x, MapSlabCoord slb_y)
```

Comment:
I think function like this is already exists somewhere. 
If I place a library it is joined with others. Same if I sell something.
These functions should be joined.

Reply:
Are you referring to the function that places a room slab?

Reply:
yes

Reply:
Where would it be?